docs/release-notes/next/http_endpoint_delete_tx_from_pool.md

```diff
@@ -0,0 +1,14 @@
+* Adds new a group of HTTP endpoints: `node-operator`. It is to hold endpoints
+  that are meant to provide functionality for tinkering with the node and
+  diving into its internals. Those should be only `internal` and the group is
+  disabled by default. The new group will be present only in the new `OAS3`
+  API and will not be present in the old `swagger2` API.
+* Adds a new HTTP endpoint: `DELETE /node/operator/mempool/hash/{hash}`. It
+  deletes a transaction currently present in the mempool. It is a part of the
+  internal `node-operator` API.
+* Fixes a bug in the transaction pool: transactions with a TTL of 0 were not
+  subject for GC of the transaction pool.
+* Fixes a bug in transaction pool: GC was triggered on syncing incoming
```

Comment:
Not sure that was a bug, we intentionally avoid extra work (like TX pool GC) when we are producing micro-blocks in `aec_conductor`... But the intuition behind that was that there were lots of different miners...